packages/ui/src/testing/sample-data/cake.json

```diff
@@ -180,6 +180,7 @@
     {
       "meta": {
         "enabled": true,
+        "valid": true,
```

Comment:
Is this necesarry? We should always read sample data through migration

Reply:
Good point. Have removed _valid_ from cake and catapult